src/auteur-widget/auteur-widget.html

```diff
@@ -22,15 +26,22 @@ <h3>${title}</h3>
       </div>
       <div class="row auteur-toevoegen-grid-container-actions">
         <div class="large-4 columns push-8">
-          <a href="${auteursUrl}/beheer/#/auteurs" target="_blank"><i class="fa fa-plus-circle"></i> Auteur aanmaken</a>
-          <a click.delegate="refresh()" class="menu-link"><i class="fa fa-refresh"></i> Vernieuwen</a>
+          <a href="${auteursUrl}/beheer/#/auteurs" target="_blank" if.bind="!isEigenaarVermogensrecht"><i class="fa fa-plus-circle"></i> Auteur aanmaken</a>
+          <a click.delegate="refresh()" class="menu-link ${isEigenaarVermogensrecht ? 'float-right' : ''}"><i class="fa fa-refresh"></i> Vernieuwen</a>
         </div>
       </div>
     </ux-dialog-body>
 
     <ux-dialog-footer style="text-align: center">
       <button class="button tiny ${buttonActief ? '' : 'disabled'}" style="margin-right: 15px;" click.delegate="addAuteur()">Toevoegen</button>
       <button class="button tiny" click.delegate="controller.cancel()">Annuleren</button>
+      <div if.bind="isEigenaarVermogensrecht" class="eigenaar-vermogensrecht-footer-text">
```

Comment:
deze 'isEigenaarVermogensrecht' check moet eigenlijk zijn: als het geen beheerder is toon link met mail, anders toon knop